app/views/session.py

```diff
@@ -13,40 +12,23 @@
 from flask import Blueprint
 from flask import url_for
 
-from flask.ext.themes2 import render_theme_template
-
 from flask_login import current_user
 
 from werkzeug.exceptions import NotFound
 
 logger = logging.getLogger(__name__)
 
 
-main_blueprint = Blueprint('main', __name__)
+session_blueprint = Blueprint('session', __name__)
 
 
-@main_blueprint.after_request
+@session_blueprint.after_request
 def add_cache_control(response):
     response.cache_control.no_cache = True
     return response
 
 
-@main_blueprint.route('/', methods=['GET'])
```

Comment:
It returns 404 currently in master, @collisdigital you may be thinking about the `/healthcheck` end point